<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BioSim&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QScrollArea" name="scrollArea_">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>258</width>
         <height>452</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="5" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="attributesLabel">
               <property name="text">
                <string>Eigensch.:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lifeLabel">
             <property name="text">
              <string>Leben.:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="velocityLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="lifeLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QTextEdit" name="attributesTextEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>89</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>89</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="velocityLabel">
             <property name="text">
              <string>Geschw.:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="strengthLabel">
             <property name="text">
              <string>St√§rke.:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="strengthLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="creatureComboBox"/>
           </item>
           <item row="6" column="2">
            <widget class="QPushButton" name="placeButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Plazieren</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Simulationssteuerung</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pauseButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stepButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Schritt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
